Makefile

```diff
@@ -0,0 +1,49 @@
+.PHONY: help fetch update cleanup dist publish
+SHELL=/bin/bash
+GEMSPEC=$(shell ls *.gemspec | head -1)
+VERSION=$(shell ruby -e 'puts Gem::Specification::load("$(GEMSPEC)").version')
+PROJECT=$(shell ruby -e 'puts Gem::Specification.load("$(GEMSPEC)").name')
+GEM=$(PROJECT)-$(VERSION).gem
+
+help: ## Print this message
+	@awk 'BEGIN { FS = ":.*##"; print "Usage:  make <target>\n\nTargets:" } \
+		/^[-_[:alpha:]]+:.?*##/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+fetch: ## Grab latest from theme repo
+	mkdir -p tmp
+	make cleanup
+	curl -o tmp/assets.zip https://codeload.github.com/MITLibraries/mitlib-style/zip/master
+	unzip -o -d tmp tmp/assets.zip
+
+update: ## Synchronize latest (run fetch_assets first)
+	# cleanup targets first in case assets were removed
+	rm -rf vendor/assets/images/
+	rm -rf vendor/assets/stylesheets/
+
+	mkdir vendor/assets/images/
+	mkdir vendor/assets/stylesheets/
+
+	cp -R tmp/mitlib-style-master/_assets/i/* vendor/assets/images
+	cp -R tmp/mitlib-style-master/_assets/sass/* vendor/assets/stylesheets
+
+	@tput setaf 2
+	@tput bold
+	@echo "Assets updated. Make sure to update version in:"
+	@echo "  lib/mitlibraries/theme/version.rb"
+	@echo "before building with 'make dist'"
+	@tput sgr0
+
+cleanup: ## Delete downloaded zip and extracted files
```

Comment:
While you can name your targets whatever you want, `clean` is traditionally one of the targets you'd find in a Makefile if you want to be all unix-y.

https://www.gnu.org/prep/standards/html_node/Standard-Targets.html